ArjunaJTS/jts/tests/classes/com/hp/mwtests/ts/jts/orbspecific/resources/AtomicWorker3.java

```diff
@@ -476,30 +478,20 @@ public static int get1() throws Exception
         try
         {
             current.begin();
-
-            try
-            {
-                returnValue = atomicObject_1.get();
-                res = true;
-            }
-            catch (TestException e)
-            {
-            }
-
-            if (res)
-                current.commit(false);
-            else
-                current.rollback();
+            returnValue = atomicObject_1.get();
+            res = true;
         }
         catch (Exception e)
         {
-            System.err.println(e);
+            logger.warn(e, e);
             throw e;
+        } finally {
+            if (res)
+                current.commit(false);
+            else
+                current.rollback();
         }
 
-        if (!res)
-            throw new Exception("Get1: Failed to retrieve value");
```

Comment:
I think if (!res) to throw an exception should be handled in the "catch throw e" above as res should never be false by this point now